apiserver/facades/client/application/service.go

```diff
@@ -44,10 +45,17 @@ type NetworkService interface {
 	GetAllSpaces(ctx context.Context) (network.SpaceInfos, error)
 }
 
-// MachineService instances save a machine to dqlite state.
+// MachineService defines the methods that the facade assumes from the Machine
+// service.
 type MachineService interface {
 	// CreateMachine creates the specified machine.
 	CreateMachine(context.Context, machine.Name) (string, error)
+	// GetMachineUUID returns the UUID of a machine identified by its name.
+	// It returns a MachineNotFound if the machine does not exist.
+	GetMachineUUID(ctx context.Context, name machine.Name) (string, error)
+	// HardwareCharacteristics returns the hardware characteristics of the
```

Comment:
**todo:** Ouch CC everywhere without review the comments. Stutting ! 

Please get ride of the double `of the` everywhere:

```txt
Matches for 'hardware characteristics of the' in [Recent Files, Types, Files, Symbols, Endpoints, Database, Config keys, Text]  (15 usages found)
    Usage in comments  (15 usages found)
        juju  (15 usages found)
            apiserver/common  (1 usage found)
                modelmanagerinterface.go  (1 usage found)
                    128 // HardwareCharacteristics returns the hardware characteristics of the
            apiserver/facades/agent/provisioner  (1 usage found)
                service.go  (1 usage found)
                    77 // HardwareCharacteristics returns the hardware characteristics of the
            apiserver/facades/agent/storageprovisioner  (1 usage found)
                service.go  (1 usage found)
                    46 // HardwareCharacteristics returns the hardware characteristics of the
            apiserver/facades/agent/uniter  (1 usage found)
                service.go  (1 usage found)
                    117 // HardwareCharacteristics returns the hardware characteristics of the
            apiserver/facades/client/application  (1 usage found)
                service.go  (1 usage found)
                    56 // HardwareCharacteristics returns the hardware characteristics of the
            apiserver/facades/client/charms  (1 usage found)
                service.go  (1 usage found)
                    38 // HardwareCharacteristics returns the hardware characteristics of the
            apiserver/facades/client/client  (1 usage found)
                service.go  (1 usage found)
                    47 // HardwareCharacteristics returns the hardware characteristics of the
            apiserver/facades/client/controller  (1 usage found)
                controller.go  (1 usage found)
                    96 // HardwareCharacteristics returns the hardware characteristics of the
            apiserver/facades/client/machinemanager  (1 usage found)
                machinemanager.go  (1 usage found)
                    111 // HardwareCharacteristics returns the hardware characteristics of the
            apiserver/facades/client/modelmanager  (1 usage found)
                services.go  (1 usage found)
                    189 // HardwareCharacteristics returns the hardware characteristics of the
            apiserver/facades/controller/firewaller  (1 usage found)
                interface.go  (1 usage found)
                    63 // HardwareCharacteristics returns the hardware characteristics of the
            apiserver/facades/controller/instancepoller  (1 usage found)
                service.go  (1 usage found)
                    44 // HardwareCharacteristics returns the hardware characteristics of the
            domain/machine/modelmigration  (1 usage found)
                export.go  (1 usage found)
                    35 // HardwareCharacteristics returns the hardware characteristics of the
            domain/machine/service  (1 usage found)
                machine_cloud_instance.go  (1 usage found)
                    37 // HardwareCharacteristics returns the hardware characteristics of the
            environs/manual/sshprovisioner  (1 usage found)
                sshprovisioner.go  (1 usage found)
                    104 // base and hardware characteristics of the remote machine
```

Reply:
Yes bad copy/paste, fixed.

Reply:
Tx.